x-pack/plugins/security_solution/public/detections/components/detection_page_filters/index.tsx

```diff
@@ -27,24 +42,21 @@ const FilterItemSetComponent = (props: FilterItemSetProps) => {
   } = useKibana();
 
   useEffect(() => {
-    // this makes sure, that if fields are not present in existing copy of the
-    // dataView, clear the cache before filter group is loaded. This is only
-    // applicable to `alert` page as new alert mappings are added when first alert
-    // is encountered
     (async () => {
-      const dataView = await dataViewService.get(dataViewId ?? '');
-      if (!dataView) return;
-      for (const filter of DEFAULT_DETECTION_PAGE_FILTERS) {
-        const fieldExists = dataView.getFieldByName(filter.fieldName);
-        if (!fieldExists) {
-          dataViewService.clearInstanceCache(dataViewId ?? '');
-          setLoadingPageFilters(false);
-          return;
-        }
-      }
+      // creates an adhoc dataview if it does not already exists just for alert index
+      const { timeFieldName = '@timestamp' } = await dataViewService.get(dataViewId ?? '');
```

Comment:
nit -> my feeling here, that we only used `@timestamp` since we control this dataview and not the user